signbank/dictionary/templates/dictionary/gloss_detail.html

```diff
@@ -171,6 +171,8 @@
 </script>
 
     <script type='text/javascript' src="{{STATIC_URL}}js/gloss_edit_color.js"></script>
+    <script type="text/javascript" src="{{STATIC_URL}}js/drag_drop_files.js"></script>
+    <script type="text/javascript" src="{{STATIC_URL}}js/recordscript.js"></script>
```

Comment:
@susanodd Could you point out where these Javascripts are used?

Reply:
Oh, I replied below. They are used in the record video routines. They were inside of a permission-prefaced block with lots of nested blocks inside it.